### PR TITLE
Fe user profile

### PR DESCRIPTION
**Pull request by LukaszBielsko**
feat: user profile

fix: refetch data after CRUD

refactor: naming change

### CODE REVIEW

File: frontend/src/authenticated/DeletePost.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
 import React from "react";
 import { deleteData } from "../utils";
+import { useDispatch } from "react-redux";
+import { saveAllPosts } from "../store/actions/actions";
 
 const DeletePost = ({ postId }) => {
+  const dispatch = useDispatch();
+
   const deletePost = () => {
     const confirmed = window.confirm("U sure?");
```
Comment (MarcinCholewka) [inline frontend/src/authenticated/DeletePost.js:10]: Instead `window confirm` you can use `https://github.com/reactjs/react-modal`

File: frontend/src/authenticated/ProfilePosts.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import DeletePost from "./DeletePost";
+import { Link } from "react-router-dom";
+
+const ProfilePosts = props => {
+  const { post } = props;
```
Comment (MarcinCholewka) [inline frontend/src/authenticated/ProfilePosts.js:6]: add empty line before return statement :-)

Comment (LukaszBielsko) [reply frontend/src/authenticated/ProfilePosts.js:6]: :)

File: frontend/src/App.css [hunk 3]
```diff
@@ -11,6 +11,52 @@
...
+    top,
+    rgba(255, 255, 255, 0.3),
+    rgba(255, 255, 255, 0)
+  );
+  background-color: #23b306;
+  border: 1px solid #3b7808;
+}
+
 .noShow {
   display: none !important;
```
Comment (MarcinCholewka) [inline frontend/src/App.css:61]: Are you sure you need that important declaration

Stephanie Rewis
> Using !important in your CSS usually means you’re narcissistic & selfish or lazy. Respect the devs to come…

https://css-tricks.com/when-using-important-is-the-right-choice/

File: frontend/src/authenticated/UserProfile.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  useEffect(() => {
+    if (user) {
+      dispatch(saveCurrentUserPosts(user.uuid));
+    }
+  }, [dispatch, user]);
+
+  return (
+    <>
+      {userPosts ? (
+        userPosts.map(post => <ProfilePosts post={post} />)
```
Comment (MarcinCholewka) [inline frontend/src/authenticated/UserProfile.js:20]: Are you sure you don't need key after mapping ???

File: frontend/src/store/actions/actions.js [hunk 5]
```diff
@@ -23,16 +25,29 @@ export const saveCurrentUser = () => async dispatch => {
...
-  const allUsers = await fetchData("/api/users");
+export const getCurrentUserPosts = currentUserPosts => ({
+  type: actionTypes.GET_CURRENT_USER_POSTS,
+  currentUserPosts
+});
 
-  dispatch(getAllUsers(allUsers));
+export const saveCurrentUserPosts = id => async dispatch => {
+  const posts = await fetchUserPosts("/api/users/", id);
+  dispatch(getCurrentUserPosts(posts));
```
Comment (MarcinCholewka) [inline frontend/src/store/actions/actions.js:35]: Add empty line before dispatch :-)

File: frontend/src/App.css (hunk 3 above)
Comment (LukaszBielsko) [reply frontend/src/App.css:61]: It is needed for hiding toolbar in ReactMD, which is external library. Cannot do it any other way.

File: frontend/src/authenticated/UserProfile.js (hunk 4 above)
Comment (LukaszBielsko) [reply frontend/src/authenticated/UserProfile.js:20]: done

File: frontend/src/App.css [hunk 6]
```diff
@@ -11,6 +11,52 @@
...
+    rgba(255, 255, 255, 0)
+  );
+  background-color: #fe1a00;
+  border: 1px solid #d83526;
+}
+
+.confirm-delete-btn {
+  background: none;
+  line-height: 1.5;
+  `font-size: 100%;
```
[the remark this answers is not in the document]
Comment (LukaszBielsko) [reply frontend/src/App.css:40]: done

**Pull request merged**